tests/system_tests_ssl.py

```diff
@@ -516,6 +592,28 @@ def test_tls_all(self):
         self.assertEqual(self.get_expected_tls_result([True, True, True, True]),
                           self.get_allowed_protocols(self.PORT_TLS_ALL))
 
+    @SkipIfNeeded(RouterTestSslBase.DISABLE_SSL_TESTING, RouterTestSslBase.DISABLE_REASON)
+    def test_tls_live(self):
+        """
+        Expects all supported versions: TLSv1, TLSv1.1, TLSv1.2 and TLSv1.3 to be allowed
+        Uses a live-created listener and profile with injected certificate files.
+        """
+        self.inject_file("ca-certificate.pem",     "live-ca.pem")
```

Comment:
We probably need one more test where the environment variable TMPDIR is set before starting the router to see if the server code that handles TMPDIR works well and a very long TMPDIR that exceeds MAX_TEMPLATE